### DISCUSSION TITLE
Welcome to MindSearch Discussions!

### DISCUSSION THREAD

**Discussion by Harold-lkk** [InternLM/MindSearch#39] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SaurabhK24** [comment]
Excited to try this out for a specific product use case - looked at the paper and looks promising!

**Comment by hedson29** [comment]
Hi 👀🫡
